lib/pages/chat_list/chat_list.dart

```diff
@@ -232,6 +232,17 @@ class ChatListController extends State<ChatList>
     });
   }
 
+  void joinInviteDirectChats() {
+    Future.wait(Matrix.of(context).client.rooms.map(joinRoom));
```

Comment:
I am not a fan of execute a heavy work inside a `StreamBuilder` or any `build` function, it is the messy between the UI and logic. Lagging in UI for sure

Reply:
and, it is not a good choice, because in the future, maybe use as spam